.rhcicd/clowdapp-backend.yaml

```
@@ -240,6 +238,12 @@ objects:
           FROM "applications" 
           INNER JOIN "endpoint_email_subscriptions" ON "applications"."id"="endpoint_email_subscriptions"."application_id" 
           GROUP BY "applications"."id" HAVING COUNT("applications"."id") > 1;
+      - prefix: insights/notifications/accounts
+        query: >-
+          SELECT
+          COUNT(distinct account_id) 
+          FROM "endpoints"
+          WHERE "endpoint_type" != 3;
```

What is the exact goal of this query?

---

Is possible to put some sort of documentation / comments to know what is each thing?

---

I can take a look at it again. But I don't think it's possible.

It's the answer for "Number of accounts that have disabled Splunk connector".

---

The current query will return the number of accounts that own at least one endpoint of any type but `CAMEL`. FYI, `splunk` is a subtype of the `CAMEL` endpoint type.

I would translate my understanding of the goal into that kind of SQL:
```suggestion
          SELECT
          COUNT(distinct account_id) 
          FROM endpoints
          WHERE endpoint_type = 3 AND endpoint_sub_type = 'splunk' AND enabled IS FALSE;
```